Create makefile to install existing binaries, header files and
pkg-config data
--- Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Makefile	2016-11-10 17:31:08.686722661 +0000
@@ -0,0 +1,23 @@
+all:	libmallincam.pc
+
+%.pc:	%.pc.in
+	sed -e "s!@LIBDIR@!$(LIBDIR)!g" \
+		-e "s!@VERSION@!$(VERSION)!g" < $< > $@
+
+install:
+	mkdir -p $(LIBDIR) $(INCDIR) $(PKGCONFDIR) $(UDEVDIR)
+	case $(DEB_BUILD_ARCH) in \
+	i386) \
+		cp $(SRCDIR)/linux/x86/libmallincam*.so.$(VERSION) $(LIBDIR);; \
+	amd64) \
+		cp $(SRCDIR)/linux/x64/libmallincam*.so.$(VERSION) $(LIBDIR);; \
+	aarch64) \
+		cp $(SRCDIR)/linux/arm64/libmallincam*.so.$(VERSION) $(LIBDIR);; \
+	armhf) \
+		cp $(SRCDIR)/linux/armhf/libmallincam*.so.$(VERSION) $(LIBDIR);; \
+	esac
+	ln -sf libmallincam.so.$(VERSION) $(LIBDIR)/libmallincam.so.$(MAJOR_REV)
+	ln -sf libmallincam.so.$(VERSION) $(LIBDIR)/libmallincam.so
+	cp $(SRCDIR)/inc/*.h $(INCDIR)
+	cp $(SRCDIR)/libmallincam.pc $(PKGCONFDIR)
+	cp $(SRCDIR)/70-mallincam-cameras.rules $(UDEVDIR)
